unittests/test_utility.py

```diff
@@ -247,7 +260,7 @@ def test_git_repo_hash(monkeypatch):
     assert osext.git_repo_hash() is None
```

Comment:
I think here we should add a case that monkeypatches the `GIT_DIR`, so as to check the case that the `rev-parse` returns an error.